lib/styles/buttons.scss

```diff
@@ -2,3 +2,17 @@
   color: red;
   cursor: pointer;
 }
+
```

Comment:
Nice! Glad we have some real sass in here. We can play around with it some more to get it looking like the spec.